djstripe/enums.py

```diff
@@ -0,0 +1,97 @@
+from enum import Enum as _Enum
+from django.utils.decorators import classproperty
+from django.utils.translation import ugettext as _
+
+
+class Enum(_Enum):
+    @classproperty
+    def choices(cls):
+        return tuple((cls.keys.get(k, k), v.value) for k, v in cls.__members__.items())
+
+    @classproperty
+    def keys(cls):
```

Comment:
I don't think we need a method like that. But I agree it's badly named... Suggestions on a better name?